internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,60 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	slice := strings.Split(data, ",")
+	if len(slice) == 2 {
```

Comment:
Лучше инвертировать условие и сделать маленький IF
```suggestion
	if len(splitData) != 2 {
		return 0, 0, fmt.Errorf("length of data is not 2")
	}
```